### DISCUSSION TITLE
daemon taking very high memory in windows

### DISCUSSION THREAD

**Discussion by nomadengineer** [Chia-Network/chia-blockchain#3469] [Support]
Hello,
The daemon.exe process is taking up too much of memory and causes plots to fail. The keeps that memory even after plotting is finished and only is released with the shutdown of GUI!

![image](https://user-images.githubusercontent.com/45670423/116827720-f56d2a80-abb3-11eb-8b42-151893b914d9.png)

One way is to farm through CLI on windows and use the GUI only for monitoring.

**Comment by nomadengineer** [comment]
As soon as the plots reach _Compressing tables 3 and 4_ its memory usage starts to rise untill it takes up too much of it!
Would it be the same if plotting done via powershell?

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
